docs/sections/contributor_guide/developer_setup.rst

```diff
@@ -1,94 +1,60 @@
-.. include:: links.rst
-
-***************
 Developer Setup
-***************
+===============
+
+.. include:: links.rst
 
 Creating a ``bash`` development shell
-=====================================
+-------------------------------------
 
 .. include:: ../miniforge3_instructions.rst
 
 2. Install the `condev`_ package into the base environment.
 
-   .. code:: sh
+  .. code:: sh
```

Comment:
I did some whitespace cleanup / consistency enforcement. Some blocks were indented e.g. 3 spaces, so I made them 2, 4, etc. (I don't think we have anything indented more than twice.)